### DISCUSSION TITLE
Page Takedowns Question

### DISCUSSION THREAD

**Discussion by MinerMinerMods** [vercel/community#5306] [Help] [answered]
Hi, I noticed that a vercel project at [downloadusdown.vercel.com](downloadusdown.vercel.com) was posing as a Minecraft download on Youtube and I was curious about how I can request vercel to take the site down due to security concerns. This appears to be my only contact method as a request takedown button doesn't appear on vercel

For Additional Info here is some provided Info
Country of Origin:
Thailand

Advertiser:
บริษัท ไบร์ท มีเดีย คอร์ปอเรชั่น จำกัด
(Bright Media Corporation Company Limited)

Ad Transparency Report:
[adstransparency.google.com/advertiser/AR12414951995310342145?origin=ata&region=anywhere](adstransparency.google.com/advertiser/AR12414951995310342145?origin=ata&region=anywhere)

AD ID:
C7AZHFdGMZcrqA6aMn88PmpehiAf7zLaDdbfTiJSAEmQQASAAYMnW5YzkpPATggEXY2EtcHViLTYyMTk4MTE3NDcwNDkzNzGgAcuGnPoCqQIPdN8oXUtfPqgDBKoE3gNP0GT00RqUfHnDRjXPzjJ2m4jT8JtDf6ivzlci7IcOZPx2nga6Fa6ZSZc7xULHSv51ksKoo7AQnWfcxSVrqdMCPhuV-q-x0dMSqRFoLSyj5fovmrmH11kaXOzEwbzT6SUAR5V4A1--hA2OSrRyiaucR62GiFby2vLlzRurOWmX7g8Lmd4ei8nJU_o-PSKyEb4qJxjoPDaZs9fhbEBMx0iNGGiREPJ7BGOOwI9shwt98-EK7da62lt7qKSufxtlzoIjauwr8Ub2rdeE6v9JsgmTpEw0VS4Lc1z8SLhE0qdmWHIQrBJ1vDs0zlLDNUJPVMu_IS8YqgSJSQO7ENWFVlksfHG0mIchTkJgi48prev6I9Hv-gw_BapMBZknOUGDk_hrcn49eZH58Fa2IAxCNNO0_P_MYJNZXE4Ul80rBEkwbbPj6Qj0UYYRZEQmvv4riOb4AtxVLCm5YaqdqXLndb6uU2H5NkansJjyfOOpnqvJ4x8bGNgGHDGMrZnJaKeuOvvWbv-bunE_hYZvphSx_bjq8hZAZjruI4sN8TrNlWD-D_64YrPJmJyAKPxNsmliRLWvJkIxKWmIWnp3Rwxr1dtd94mPbV5Rl5b_bAIYP7F9ZB22goV-pEmErPyyzdXriAXf1p32S6AGVNgGAoAHnfnjhQGIBwGQBwKYBwKoB_mxsQKoB-edsQKoB-idsQKoB-q3sQKoB6e1sQKoB--1sQKoB_C1sQKoB4KqsQKoB4QIqAeo0huoB7YHqAekmrECqAeRn7ECqAfmmrECqAf0mbECqAewm7ECqAffobECqAeusbECqAfmvLECqAemqrECqAf9srECqAf3uLECqAf4uLECqAeBxhuoB_yvsQKoB-PZG5IICzdycS1yY2JubmJVqAgBsAgMuAgB0gglCITAgMAEEAIYWjIEgcaADjoMwtCAgICARIDAgIAgSNmg0jVQFPIIH2NhLXl0LWhvc3QtcHViLTUwNzI5NDkxNzI0NTIxOTLICRjICY8ByAmQAaELoO4PAQQ3Piu6C18IARABGAUgASgBMAtABUgCYABoAHACiAEAmAEBogE3Ch8IASACKALauAQVMhM6EQoPKDBIp8-z2vkYmBkBuCABGAAgAFgAYABwAIABAYgCAqgCBcACAtgBAYACAYgCBtIMBAgBEAG4E____________wGIFAKwFAHAFcmAgECQFgGKFxgIAxgBIAEoATABOAFAAUgBUAFYAWACcAGgFwGpF9LFrF3nK6-quhcGEAEoADAB

**Comment by swarnava** [comment, marked as answer]
Hey, If you believe that a Vercel deployment contains malicious or illegal content, please email [abuse@vercel.com](mailto:abuse@vercel.com). When doing this, please provide as much relevant information as possible, taking into consideration our [DMCA policy](https://vercel.com/legal/dmca-policy) and [trademark policy](https://vercel.com/legal/trademark-policy).
